scripts/run_sim.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
Keeping this as `#!/bin/sh` is fine. It's only bash that we want it to be `#!/usr/bin/env bash`